Project2015To2017.Core/Project2015To2017.Core.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.3;netstandard2.0</TargetFrameworks>
+    <RootNamespace>Project2015To2017</RootNamespace>
```

Comment:
We don't need to do this now but I like having the namespaces line up with the project names.

Reply:
Me too, but that's for maintaining (mostly) stable public API surface.